sample/src/main/kotlin/com/slack/circuit/sample/petlist/PetList.kt

```diff
@@ -88,26 +88,24 @@ import javax.inject.Inject
 import kotlinx.coroutines.flow.Flow
 import kotlinx.parcelize.Parcelize
 
-@Immutable
-@Parcelize
 data class PetListAnimal(
   val id: Long,
   val name: String,
   val imageUrl: String?,
   val breed: String?,
   val gender: String,
   val age: String,
-) : Parcelable
+)
 
 @Parcelize
 object PetListScreen : Screen {
-  sealed interface State : Parcelable {
-    @Parcelize object Loading : State
-    @Parcelize object NoAnimals : State
-    @Parcelize data class Success(val animals: List<PetListAnimal>) : State
```

Comment:
Do we not need/want these to be `Parcelable`?

Reply:
🤔 never mind, I guess we aren't using `rememberSaveable` making use of parcelable pointless